### PR TITLE
assignment 2 Akshay jha

### PR DESCRIPTION
**Pull request by Akshay666**

### CODE REVIEW

**Comment by Akshay666** [comment]
Fixing Readme issues.

**Comment by amfleming** [comment]
Unmerge-able due to conflicts

**Comment by Akshay666** [comment]
Fixing ASAP.

File: README.md [hunk 1]
```diff
@@ -65,6 +65,36 @@ People
 * [Marco Berlot](people/marco-berlot.md) - mb2589 - [http://ec2-52-202-201-59.compute-1.amazonaws.com:8080](http://ec2-52-202-201-59.compute-1.amazonaws.com:8080)[![CircleCI](https://circleci.com/gh/MarcoBerlot/dev_skeleton.svg?style=svg)](https://circleci.com/gh/MarcoBerlot/dev_skeleton)
 * [Stephen Bongner](people/stephen-bongner.md) - sb2483 - [http://ec2-13-59-91-251.us-east-2.compute.amazonaws.com:8080](http://ec2-13-59-91-251.us-east-2.compute.amazonaws.com:8080)[![CircleCI](https://circleci.com/gh/sb2483/skeleton.svg?style=svg)](https://circleci.com/gh/sb2483/skeleton)
 * [Cameron Boroumand](people/cameron-boroumand.md) - cb596 - [http://cameronboroumand.com.s3-website.us-east-2.amazonaws.com/grading.html](http://cameronboroumand.com.s3-website.us-east-2.amazonaws.com/grading.html)
+* [Gaurav Chaturvedi](people/gaurav-chaturvedi.md) - gkc24 - [http://gkchat.s3-website.us-east-2.amazonaws.com/grading.html](http://gkchat.s3-website.us-east-2.amazonaws.com/grading.html)
```
Comment (amfleming) [inline README.md]: What are all these additions?

**Review by amfleming — CHANGES_REQUESTED** [review]

**Pull request closed without merging**